README.md

```diff
@@ -1,13 +1,13 @@
+![Drupal Commerce](https://drupalcommerce.org/sites/default/files/dclogo_stacked_2c_on_white.png "Drupal Commerce logo")
```

Comment:
I'd recommend to download this image into the `icons` folder. So, the markdown will be the following:

\!\[Drupal Commerce\]\(icons/dclogo_stacked_2c_on_white.png "Drupal Commerce logo")

You may see the example how it's down on the [Commerce Extended Attributes](https://github.com/drugan/commerce_xattributes) module's [README.md](https://raw.githubusercontent.com/drugan/commerce_xattributes/8.x-1.x/README.md) file.